src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -461,6 +476,8 @@ private void SetupScene()
                 Position = new Point3D(10, 15, 10),
                 LookDirection = new Vector3D(-10, -10, -10),
                 UpDirection = new Vector3D(0, 1, 0),
+                NearPlaneDistance = 1,
+                FarPlaneDistance = 2000000,
```

Comment:
As discussed, this is not a windows media ProjectionCamera, it's a Helix ProjectionCamera which manages its own matrices. When the clip plane is set to very large numbers, there is a division by NaN. This needs further investigation.